travis_ci/test_docs.bash

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# based on https://stackoverflow.com/a/49516361/7728169
+
+set -e
+
+if [ -n "$TRAVIS_TAG" ]; then
+  # this is a tag build
+  export DOCS_VERSION="$TRAVIS_TAG"
+elif [[ -n "$TRAVIS_BRANCH" && "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+  export DOCS_VERSION="$TRAVIS_BRANCH"
+else
+  # nothing to deploy
+  exit 0
```

Comment:
If this isn't a tag, release or PR merge, we don't update the documentation on the web.